t/apicast-policy-standalone.t

```diff
@@ -199,3 +199,20 @@ GET /t HTTP/1.1
 --- no_error_log
 [error]
 [warn]
+
+
+
+=== TEST 6: invalid configuration
+--- environment_file: standalone
+--- SKIP: works on OpenResty 1.15.x
```

Comment:
I don't understand this.

Reply:
I'm testing this on OpenResty 1.15 locally and it works but does not work on the CI (1.13).

We should remove the SKIP once we are on 1.15. They changed passing signals between master/worker and it's shutdown.

Reply:
Thanks for the clarification.